os-eap-txnrecovery/added/partitionPV.sh

```diff
@@ -37,6 +37,9 @@ function arrContains() {
 function init_pod_name() {
   # when POD_NAME is non-zero length using that given name
 
+  # if the user specifies NODE_NAME or JBOSS_NODE_NAME we use that this is generally used only by tests
```

Comment:
Is this "generally used only by" or "only meant to be used by"?

The former is ok if the latter is not true. Otherwise be blunt. ;)

Reply:
Its really 'only meant to be used by', I was trying to be nice :) Will update.